### ISSUE TITLE
Test failure: `com.linecorp.armeria.xds.client.endpoint.RampingUpTest.checkEndpointsAreRampedUp()`

### ISSUE THREAD

**Issue by minwoox** [line/armeria#5749]
```
java.lang.AssertionError: 
Expecting actual:
  [null,
    Endpoint{a.com:80, weight=1, attributes=[com.linecorp.armeria.xds.client.endpoint.XdsAttributeKeys#LOCALITY_LB_ENDPOINTS_KEY=locality {
}
lb_endpoints {
  endpoint {
    address {
      socket_address {
        address: "a.com"
        port_value: 80
      }
    }
  }
  health_status: HEALTHY
  metadata {
  }
  load_balancing_weight {
    value: 10
  }
}
lb_endpoints {
  endpoint {
    address {
      socket_address {
        address: "b.com"
        port_value: 80
      }
    }
  }
  health_status: HEALTHY
  metadata {
  }
  load_balancing_weight {
    value: 10
  }
}
, com.linecorp.armeria.internal.client.endpoint.RampingUpKeys#createdAtNanos=31975871750669554, com.linecorp.armeria.xds.client.endpoint.XdsAttributeKeys#LB_ENDPOINT_KEY=endpoint {
  address {
    socket_address {
      address: "a.com"
      port_value: 80
    }
  }
}
health_status: HEALTHY
metadata {
}
load_balancing_weight {
  value: 10
}
]},
    Endpoint{b.com:80, weight=1, attributes=[com.linecorp.armeria.xds.client.endpoint.XdsAttributeKeys#LOCALITY_LB_ENDPOINTS_KEY=locality {
}
lb_endpoints {
  endpoint {
    address {
      socket_address {
        address: "a.com"
        port_value: 80
      }
    }
  }
  health_status: HEALTHY
  metadata {
  }
  load_balancing_weight {
    value: 10
  }
}
lb_endpoints {
  endpoint {
    address {
      socket_address {
        address: "b.com"
        port_value: 80
      }
    }
  }
  health_status: HEALTHY
  metadata {
  }
  load_balancing_weight {
    value: 10
  }
}
, com.linecorp.armeria.internal.client.endpoint.RampingUpKeys#createdAtNanos=31975871750669554, com.linecorp.armeria.xds.client.endpoint.XdsAttributeKeys#LB_ENDPOINT_KEY=endpoint {
  address {
    socket_address {
      address: "b.com"
      port_value: 80
    }
  }
}
health_status: HEALTHY
metadata {
}
load_balancing_weight {
  value: 10
}
]}]
to contain exactly in any order:
  [Endpoint{a.com:80, weight=1000}, Endpoint{b.com:80, weight=1000}]
but the following elements were unexpected:
  [null]

	at com.linecorp.armeria.xds.client.endpoint.RampingUpTest.checkEndpointsAreRampedUp(RampingUpTest.java:137)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at java.util.ArrayList.forEach(ArrayList.java:1259)
	at java.util.ArrayList.forEach(ArrayList.java:1259)

```

**Comment by minwoox** [comment]
It happened again. 🤔 
https://github.com/line/armeria/actions/runs/9790150764/job/27031187815?pr=5796#step:11:2473

**Comment by minwoox** [comment]
Closed by #5799
